### PR TITLE
chore(npm): organize NPM dependencies

### PR DESCRIPTION
**Pull request by daniellacosse**
I'm moving some things to `devDependencies` that should be there and removing stuff that seems to be no longer in use

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -94,26 +86,27 @@
...
+    "globby": "^13.1.1",
     "gulp": "^4.0.0",
     "html-webpack-plugin": "^5.1.0",
     "husky": "^1.3.1",
+    "intl-messageformat": "^9.12.0",
     "karma": "^6.3.16",
     "karma-chrome-launcher": "^3.1.0",
     "karma-jasmine": "^4.0.1",
     "karma-webpack": "^5.0.0",
+    "lit": "^2.2.2",
```
Comment (fortuna) [inline package.json:98]: Why is this moving to dev? It's a direct dependency of our web code.

File: package.json [hunk 2]
```diff
@@ -94,26 +86,27 @@
     "electron-icon-maker": "^0.0.4",
     "esm": "^3.2.25",
     "fancy-log": "^1.3.3",
+    "globby": "^13.1.1",
     "gulp": "^4.0.0",
     "html-webpack-plugin": "^5.1.0",
     "husky": "^1.3.1",
+    "intl-messageformat": "^9.12.0",
```
Comment (fortuna) [inline package.json:93]: We use this in our web code.

**Review by fortuna — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (daniellacosse) [reply package.json:93]: Ah, duh

File: package.json (hunk 1 above)
Comment (daniellacosse) [reply package.json:98]: At this moment we're only using it in storybook.

File: package.json (hunk 2 above)
Comment (daniellacosse) [reply package.json:93]: Right now the only import I see is in the storybook:

https://github.com/Jigsaw-Code/outline-client/blob/8016293e1a0f7b58754f076359065a9670c37434/src/www/.storybook/localize.ts#L18-L19

Comment (fortuna) [reply package.json:93]: You're right. We use it in the Manager, I think

**Pull request merged**